python/ray/tune/trainable.py

```diff
@@ -205,6 +205,31 @@ def train(self):
 
         return result
 
+    def save_checkpoint_relative(self, relative_dir):
+        """Saves checkpoints relative to logdir folder.
+
+        Parameters
+        ----------
+            relative_dir : path of subdirectory to save to
+
+        Returns
+        -------
+            checkpoint_path, checkpoint_folder_path
+
+        """
+        folder_path = os.path.join(self.logdir, relative_dir)
+        return self.save(folder_path), os.path.join(folder_path, "checkpoint_{}".format(self._iteration))
```

Comment:
I don't understand what are you doing here with returning two values and creating checkpoint dir folder manually? save already does that and returns that value?

Reply:
save returns filepath and the other one is folderpath. Alternative is just to strip last part of the path returned by save

Reply:
I would say that finding folder of returned path is a less assumption than checkpoint name will certainly be  "checkpoint_{}".format(self._iteration)